kclvm/api/src/service/capi.rs

```diff
@@ -33,25 +34,23 @@ pub extern "C" fn kclvm_service_free_string(res: *mut c_char) {
 macro_rules! call {
     ($serv:expr, $args:expr, $arg_name:ident, $serv_name:ident) => {{
         let serv_ref = unsafe { mut_ptr_as_ref($serv) };
-        let args = $arg_name::parse_from_bytes($args).unwrap();
+        let args = unsafe { std::ffi::CStr::from_ptr($args) }.to_bytes();
+        let args = $arg_name::decode(args).unwrap();
         let res = serv_ref.$serv_name(&args);
         let result_byte = match res {
-            Ok(res) => match res.write_to_bytes() {
-                Ok(bytes) => bytes,
-                Err(err) => panic!("{err}"),
-            },
-            Err(err) => panic!("{err}"),
+            Ok(res) => res.encode_to_vec(),
+            Err(err) => panic!("{}", err),
```

Comment:
U can run `cargo clippy --fix --allow_dirty` in the kclvm api crate to fix the code
```
panic!("{}", err) -> panic!("{err}") 
```